netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
We are trying to do away with run-test.sh and switch to build.sh. Do we really want to go to a place where we depend on hacking the various drops together? It seems very fragile and error prone. How much value do we think running against the checked version of coreclr in our repro adds?